test/components/charts/lChartGrid.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { screen } from '@testing-library/vue'
+import { composeStories } from '@storybook/testing-vue'
+import { renderComponent } from '~/test/utils.setup.testingLibrary'
+import * as stories from '~/docs/stories/components/charts/LChartGrid.stories'
+
+const { Default } = composeStories(stories)
+
+describe('LResumeChart', () => {
+  it('should contain the side titles', () => {
```

Comment:
Ao invés de "contain" eu usaria o termo "render", "should render the side titles", mas se não achar pertinente dá pra manter assim.